src/hdf5plugin/test.py

```diff
@@ -438,6 +438,7 @@ def testSelection(self):
 class TestSZ(unittest.TestCase):
     """Specific tests for SZ compression"""
 
+    @unittest.skipUnless(should_test("sz"), "SZ filter not available")
```

Comment:
This was missing and it's convenient to have for testing without compiling all filters